[2025-05-27T21:06:01.270+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hello_airflow.say_hello scheduled__2025-05-27T21:05:00+00:00 [queued]>
[2025-05-27T21:06:01.274+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hello_airflow.say_hello scheduled__2025-05-27T21:05:00+00:00 [queued]>
[2025-05-27T21:06:01.274+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-27T21:06:01.281+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): say_hello> on 2025-05-27 21:05:00+00:00
[2025-05-27T21:06:01.288+0000] {standard_task_runner.py:57} INFO - Started process 209 to run task
[2025-05-27T21:06:01.291+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'hello_***', 'say_hello', 'scheduled__2025-05-27T21:05:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/example_dag.py', '--cfg-path', '/tmp/tmp2rwii4ra']
[2025-05-27T21:06:01.293+0000] {standard_task_runner.py:85} INFO - Job 7: Subtask say_hello
[2025-05-27T21:06:01.315+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:193 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-27T21:06:01.338+0000] {task_command.py:416} INFO - Running <TaskInstance: hello_airflow.say_hello scheduled__2025-05-27T21:05:00+00:00 [running]> on host d904dd6e076e
[2025-05-27T21:06:01.407+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='hello_***' AIRFLOW_CTX_TASK_ID='say_hello' AIRFLOW_CTX_EXECUTION_DATE='2025-05-27T21:05:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-27T21:05:00+00:00'
[2025-05-27T21:06:01.409+0000] {logging_mixin.py:151} INFO - Hello from Airflow!
[2025-05-27T21:06:01.409+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-27T21:06:01.415+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=hello_***, task_id=say_hello, execution_date=20250527T210500, start_date=20250527T210601, end_date=20250527T210601
[2025-05-27T21:06:01.435+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-27T21:06:01.442+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
